### PR TITLE
Chore/add miw and vault

### PR DESCRIPTION
**Pull request by ds-jhartmann**
## Description

- Added MIW running configuration
- Added Vault
- Added secret seeding postinstall job for vault

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by evegufy — COMMENTED** [review]
@ds-jhartmann I tested the change locally and when not using the keycloak from the miw but the centralidp instead the install is successful:
https://github.com/eclipse-tractusx/tractus-x-umbrella/commit/c84c06a7b800cb3a01061738de027579247133b3
https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/runs/8379432620

**Comment by ds-jhartmann** [comment]
> @ds-jhartmann I tested the change locally and when not using the keycloak from the miw but the centralidp instead the install is successful: [c84c06a](https://github.com/eclipse-tractusx/tractus-x-umbrella/commit/c84c06a7b800cb3a01061738de027579247133b3) https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/runs/8379432620

Thanks a lot, I added your suggestion :)

File: charts/umbrella/templates/post-install-vault-setup.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{{- if .Values.vault.enabled }}
```
Comment (evegufy) [inline charts/umbrella/templates/post-install-vault-setup.yaml:1]: could you please still add a file header?

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/umbrella/templates/post-install-vault-setup.yaml (hunk 1 above)
Comment (ds-jhartmann) [reply charts/umbrella/templates/post-install-vault-setup.yaml:20]: done

Comment (evegufy) [reply charts/umbrella/templates/post-install-vault-setup.yaml:20]: the linter failed unfortunately

Comment (ds-jhartmann) [reply charts/umbrella/templates/post-install-vault-setup.yaml]: fixed

**Pull request merged**